src/views/HomeView.vue

```diff
@@ -104,16 +112,17 @@ const sportsWithIcon = computed(() => allSports.value.map(sport => ({
 })))
 
 // // Submit button will have: 'eventId', 'betId', 'teamId'
-function sendBet(eventID, teamID, betObj){
-const bet = {
-        betId: betObj.id,
-        eventId: eventID,
-        teamId: teamID };
-
-axios.post('/submit', bet)
-  .then((result) => (console.log(result.data)))
+const sendBet = (eventID, teamID, betID) => {
+  const bet = {
+          betId: betID,
+          eventId: eventID,
+          teamId: teamID };
+
+  axios.post('/submit', bet)
+    .then((result) => (console.log(result.data)))
```

Comment:
indents look a little off here.

Also on line 122, you can use deconstruction really nicely: `.then(({ data }) => console.log(data))`

Reply:
Modified to use deconstruction